### ISSUE TITLE
[vcpkg] 2024.07.12 Release Notes are referring to the wrong vcpkg-tool version

### ISSUE THREAD

**Issue by petamas** [microsoft/vcpkg#39951]
**Describe the bug**
The [Release Notes for 2024.07.12](https://github.com/microsoft/vcpkg/releases/tag/2024.07.12) says that `vcpkg-tool` 2024-06-10 has been released since the last `vcpkg` release, but that's the same version the [Release Notes for `vcpkg` 2024.06.15](https://github.com/microsoft/vcpkg/releases/tag/2024.06.15) mentioned. Also, the last commit in 2024.07.12 is https://github.com/microsoft/vcpkg/pull/39836, which pulls in `vcpkg-tool` 2024-07-10.

TLDR: `vcpkg` 2024.07.12 release notes has a typo in the `vcpkg-tool` version number, referring to 2024-06-10 instead of 2024-07-10.

**Comment by LilyWangLL** [comment]
Hello, I did not see the 2024-06-10 note in the release interface for 2024.07.12, as shown in the following image:
![image](https://github.com/user-attachments/assets/97c24695-f221-449e-b2a8-781a9343c94a)

**Comment by petamas** [comment]
Seems like someone has already fixed it. Feel free to close this issue.
